wrote content for how to write zkapp page without screenshot (#284)
o1-labs/docs2: docs/using-mina/how-to-use-zkapp.mdx

title: How to use a zkApp
---

import Subhead from '@site/src/components/common/Subhead';

# How to Use a zkApp

<Subhead>Learn how to interact with a zero-knowledge smart contract</Subhead>

:::info

Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.

:::

### Prerequisites

1. Install a zkApp-compatible [Mina wallet](../using-mina/install-a-wallet).
2. Make sure that your Mina wallet contains MINA to pay for transaction fees. 

:::note

> The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps.

Reviewer: ```suggestion
The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps currently.
```

:::

### Instructions

1. Visit the zkApp in a web browser. For example, `mycoolzkapp.com`.
2. Interact with the zkApp as intended. For example, make a move in a game, enter in your age, and so on.
3. Click the confirmation button to send the transaction to the Mina network.
4. In your Mina browser wallet extension, confirm the transaction.

> 5. Done! Congratulations. Your transaction is processed by the Mina network and the zkApp state is updated.

Reviewer: ```suggestion
5. Done! Congratulations. Your transaction will be processed by the Mina network and, when accepted into a block, will update the zkApp's on-chain state.
```

Reviewer: We need to use future tense here b/c the transaction being sent to the network sits there until processed into a block. Only if/when it is processed into a block, will it update the zkApp account on chain.

title: How to use a zkApp
---

import Subhead from '@site/src/components/common/Subhead';

# Install a Wallet

<Subhead>Learn how to interact with a zero-knowledge smart contract</Subhead>

> :::info

Reviewer: Should appear above the H1, for consistency with other pages.

Reviewer 2: @iregina  I see now... the zkApp docs have more solid examples of placement for the header

see https://github.com/o1-labs/docs2/blob/main/docs/zkapps/how-to-test-a-zkapp.mdx for a good example, move to line 6

Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.

:::

### Prerequisites

> To use a zkApp:

Reviewer: ```suggestion
```
testing out the inline commit suggestion

> 1. Install a Chrome zkApp-compatible Mina wallet.

Reviewer: +1 for removing `Chrome` here

2. Make sure that your Mina wallet contains MINA to pay for transaction fees. 

:::note

The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps.

:::

### Instructions

> 1. Visit the zkApp in a web browser. For example,  mycoolzkapp.com.

Reviewer: +1

2. Interact with the zkApp as intended. For example, make a move in a game, enter in your age, and so on.
3. Click the confirmation button to send the transaction to the Mina network.

> 4. In your Mina browser wallet extension, confirm the transaction again.

Reviewer: -again 

They're only confirming it once. The button on the page starts that interactions, but they're actually confirming via the wallet.
